appveyor.yml

```diff
@@ -0,0 +1,144 @@
+environment:
+  TWINE_USERNAME: bauerj
```

Comment:
Why is my username in there? 🔍 🤔

Reply:
Because this commit is only testing your fork and fix some error.